vaadin-platform-react-hybrid-test/src/main/java/com/vaadin/platform/react/test/views/FlowMainView.java

```diff
@@ -0,0 +1,27 @@
+package com.vaadin.platform.react.test.views;
+
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.html.Span;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.router.Route;
+import com.vaadin.flow.router.RouterLink;
+
+@Route("flow")
+public class FlowMainView extends VerticalLayout {
```

Comment:
I don't know why this doesn't happening in the validation of this PR, but locally I get 
```
Caused by: com.vaadin.flow.server.InvalidRouteConfigurationException: Invalid route configuration. The following Hilla route(s) conflict with configured Flow routes: flow
```
because this view has apparently the same mapping as `views/flow/@index.tsx`.

Reply:
Even if I remove `views/flow/@index.tsx`, it still throws, because apparently it sees an entry with "flow" path in the `file-routes.json`.

Reply:
I fixed it by using "flow-flow" mapping for Flow view.

Reply:
There was the check for duplicate routes PR just a while ago and this one is older than that.
But just removing the `@index.tsx` from the flow folder should be enough. Probably the mapping check needs some change on this.

Reply:
Just removing it doesn't help. Perhaps we have a bug in conflicts detection in Flow.

Reply:
Ok, validation passes.